I'm sorry, but an uncaught exception occurred.

While running game code:
  File "renpy/common/00start.rpy", line 244, in script call
    call _splashscreen from _call_splashscreen_1
  File "game/security.rpy", line 83, in script
    text "[message]\n\nLe jeu va se fermer." with dissolve
Exception: Sayer 'text' is not defined.

-- Full Traceback ------------------------------------------------------------

Full traceback:
  File "renpy/common/00start.rpy", line 244, in script call
    call _splashscreen from _call_splashscreen_1
  File "game/security.rpy", line 83, in script
    text "[message]\n\nLe jeu va se fermer." with dissolve
  File "C:\Users\Aris\Documents\renpy-8.3.7-sdk\renpy\ast.py", line 2586, in execute
    Say.execute(self)
  File "C:\Users\Aris\Documents\renpy-8.3.7-sdk\renpy\ast.py", line 583, in execute
    who = eval_who(self.who, self.who_fast)
  File "C:\Users\Aris\Documents\renpy-8.3.7-sdk\renpy\ast.py", line 472, in eval_who
    raise Exception("Sayer '%s' is not defined." % who)
Exception: Sayer 'text' is not defined.

Windows-10-10.0.26100 AMD64
Ren'Py 8.3.7.25031702
Arisization Project Version_0.0.5
Sun Aug 24 00:06:34 2025
